Giovaniavs/MyProject-server
# Feature/base server express

## What I did
* Created the base server of the project.

## How to test
1. Run the following command in the terminal of the project: **yarn start** or **npm start** if you're using the npm.
1.1. It should show in the terminal a message **Server runing on port 3001**
